application-templates/starter/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "merchant-center-application-template-starter",
-  "version": "12.2.0",
+  "version": "12.0.1-pre-canary.32",
```

Comment:
Q: should we revert those version before merging? I guess it shouldn't matter if we then release a major, all version will be bumped anyway.

Reply:
What do you mean by revert? Unpublishing won't work after 24 hours. We could remove the dist-tag.